src/platform/packages/shared/kbn-esql-validation-autocomplete/scripts/generate_function_definitions.ts

```diff
@@ -692,6 +692,29 @@ const enrichGrouping = (
   });
 };
 
+/** Enriches scalar functions that only accept a query parameter */
+const enrichQueryStringFunction = (functionDefinition: FunctionDefinition): FunctionDefinition => {
+  if (
+    functionDefinition.signatures.every(
+      (s) => s.params.length > 0 && s.params.every((p) => p.name === 'query')
```

Comment:
Probably not too dangerous, but we can't really count on these parameter names. ES team could change them at any time.

Reply:
changed to functionEnrichments in https://github.com/elastic/kibana/commit/6b0a6b08d7c79b2193a927e183bec226bbd50932